Files/Strings/en-US/Resources.resw

```diff
@@ -1950,5 +1953,49 @@
     <data name="UnpinItemFromStart.Text" xml:space="preserve">
     <value>Unpin from the Start Menu</value>
   </data>
-  
+  <data name="ItemTypeLibrary" xml:space="preserve">
+    <value>Library</value>
+  </data>
+  <data name="LibraryItemAutomation" xml:space="preserve">
+    <value>Library</value>
+  </data>
+  <data name="LibraryNoLocations" xml:space="preserve">
+    <value>No locations</value>
+  </data>
+  <data name="LibraryCardsCreateNewLibraryInputPlaceholderText" xml:space="preserve">
+    <value>Desired name</value>
+  </data>
+  <data name="LibraryCardsCreateNewLibraryDialogTitleText" xml:space="preserve">
+    <value>Create Library</value>
+  </data>
+  <data name="LibraryCardsCreateNewLibraryDialogSubtitleText" xml:space="preserve">
+    <value>Enter the name of the new library</value>
+  </data>
+  <data name="ErrorInputEmpty" xml:space="preserve">
+    <value>Input field cannot be empty!</value>
+  </data>
+  <data name="ErrorNameInputRestrictedCharacters" xml:space="preserve">
+    <value>The name must not contain the following characters: \ / : * ? " &lt; &gt; |</value>
+  </data>
+  <data name="ErrorNameInputRestricted" xml:space="preserve">
+    <value>Using this name is not allowed!</value>
+  </data>
+  <data name="CreateLibraryErrorAlreadyExists" xml:space="preserve">
+    <value>Library with the same name already exists!</value>
+  </data>
+  <data name="LibraryCardsDeleteLibraryDialogTitleText" xml:space="preserve">
+    <value>Delete Library</value>
+  </data>
+  <data name="LibraryCardsDeleteLibraryDialogSubtitleText" xml:space="preserve">
+    <value>Do you really want to delete this library? The files and folders will remain on your filesystem.</value>
+  </data>
+  <data name="DialogCreateButtonText" xml:space="preserve">
+    <value>Create</value>
+  </data>
+  <data name="DialogCancelButtonText" xml:space="preserve">
+    <value>Cancel</value>
+  </data>
+  <data name="DialogDeleteButtonText" xml:space="preserve">
+    <value>Delete</value>
+  </data>
```

Comment:
```suggestion
  <data name="DialogCreateLibraryButtonText" xml:space="preserve">
    <value>Create</value>
  </data>
  <data name="DialogCancelButtonText" xml:space="preserve">
    <value>Cancel</value>
  </data>
  <data name="DialogDeleteLibraryButtonText" xml:space="preserve">
    <value>Delete</value>
  </data>
```

Reply:
@yaichenbaum I had the sudden idea to use a single entry for all the common `DynamicDialog` buttons (Cancel, Create, Delete, etc.) in the future and not a different one to each. Though this might help to reduce the size of the translation file should be in a different PR, so I'll update string names here.

Reply:
@nvi9 Sounds good!